### PR TITLE
localized number at thank you page

### LINKED ISSUE #274: Localize Number at Thank You Page
Screen Shot 2020-09-26 at 4 44 34 PM

### PR DESCRIPTION
**Pull request by captainamiedi**
Fixes #274 

Changes in this pull request:
-
-
-

@sagararyal 
localized

localized2
localized3

### CODE REVIEW

**Comment by sagararyal** [comment]
- [ ] Please localize the amount at step 2 of donation process as well

Screen Shot 2020-10-12 at 11 29 28 PM

File: public/static/locales/en/donate.json [hunk 1]
```diff
@@ -58,5 +58,6 @@
   "noProjectsFound": "No projects found",
   "inTimeOfTaxReturns": "in time for tax returns.",
   "titleToShare": "Planting trees against the climate crisis!",
-  "textToShare": "Preventing the climate crisis requires drastically reducing carbon emissions and planting trees. That’s why I just planted some. Check out {{linkToShare}} if you want to plant some too! #StopTalkingStartPlanting #TrillionTrees @PftP_int"
+  "textToShare": "Preventing the climate crisis requires drastically reducing carbon emissions and planting trees. That’s why I just planted some. Check out {{linkToShare}} if you want to plant some too! #StopTalkingStartPlanting #TrillionTrees @PftP_int",
+  "fortreeAmountTrees": "{{amount}}"
```
Comment (norbertschuler) [inline public/static/locales/en/donate.json:62]: Adding a new translation resource which contains just a variable is not necessary and just add unnecessary complexity to our code (and more work for translators).

File: src/features/public/Donations/components/ContactDetails.tsx [hunk 2]
```diff
@@ -214,7 +215,10 @@ function ContactDetails({
 
         <div className={styles.finalTreeCount}>
           <div className={styles.totalCost}>
-            {currency} {Sugar.Number.format(Number(treeCount * treeCost), 2)}
+          {t('donate:fortreeAmountTrees', {
+            amount: getFormatedCurrency(i18n.language, currency, treeCount * treeCost),
```
Comment (norbertschuler) [inline src/features/public/Donations/components/ContactDetails.tsx:219]: `{ getFormatedCurrency(i18n.language, currency, treeCount * treeCost) }` should be enough here?

**Review by norbertschuler — CHANGES_REQUESTED** [review]

File: src/features/public/Donations/components/ContactDetails.tsx (hunk 2 above)
Comment (captainamiedi) [reply src/features/public/Donations/components/ContactDetails.tsx]: ok. i have made the changes

**Comment by sagararyal** [comment]
@captainamiedi donate step 3 is not completed.
Screen Shot 2020-10-14 at 3 10 54 PM

**Comment by captainamiedi** [comment]
> @captainamiedi donate step 3 is not completed.
> Screen Shot 2020-10-14 at 3 10 54 PM

I have updated it. You can test now

**Review by harshvitra — APPROVED** [review]

File: public/static/locales/en/donate.json (hunk 1 above)
Comment (harshvitra) [reply public/static/locales/en/donate.json]: This is resolved

**Pull request merged**